### PR TITLE
chore: Create WDS Text Widget

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #28210

### CODE REVIEW

File: app/client/src/widgets/wds/WDSTextWidget/config/defaultsConfig.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import type { WidgetProps } from "widgets/BaseWidget";
+import { isDynamicValue } from "utils/DynamicBindingUtils";
+import type { DynamicPath } from "utils/DynamicBindingUtils";
+import { FILL_WIDGET_MIN_WIDTH } from "constants/minWidthConstants";
+import { BlueprintOperationTypes } from "WidgetProvider/constants";
+import { ResponsiveBehavior } from "layoutSystems/common/utils/constants";
+
+export const defaultsConfig = {
+  text: "Hello {{appsmith.user.name || appsmith.user.email}}",
+  fontSize: "body",
```
Comment (KelvinOm) [inline app/client/src/widgets/wds/WDSTextWidget/config/defaultsConfig.ts:11]: Do these options affect our component somehow?

```
fontSize: "body",
fontStyle: "bold",
textAlign: "left",
textColor: "neutral",
```

**Review by KelvinOm — COMMENTED** [review]
LGTM. One moment, why is the text bold by default?

Снимок экрана 2023-10-19 в 10 38 13

**Comment by jsartisan** [comment]
> LGTM. One moment, why is the text bold by default?
It is there in the current text widget, so I copied the default value

@KelvinOm

**Comment by shadabbuchh** [comment]
Bugs in this PR:

1. Although the `Expected structure` says `String`, on entering a value starting with upper case for `Font size`, `Text color` & `Emphasis` do not render appropriately. For `Alignment` both values are accepted. Can we please make this consistent?

https://github.com/appsmithorg/appsmith/assets/39921438/0bbeb119-5b96-425f-9ef4-c68e9aa8e3da

2. Upon entering `0` in `Line clamp` nothing happens (No error or anything)
Screenshot 2023-10-20 at 9 27 12 AM

3. No avenue for overflow text: (Thought of calling it out, but is this relevant since auto height will not be available in Anvil?)

https://github.com/appsmithorg/appsmith/assets/39921438/baf4354f-9adc-4287-b677-9a00dc103d0e

**Comment by shadabbuchh** [comment]
We still do not throw any error when `Bold/Italic` is entered for `Emphasis`. Created a separate issue for this: https://github.com/appsmithorg/appsmith/issues/28252
All other reported issues are fixed except the 3rd one as we won't have the `Height` property in anvil.

**Pull request merged**